src/ipc/handler.c

```diff
@@ -1245,8 +1245,9 @@ int ipc_queue_host_message(struct ipc *ipc, uint32_t header, void *tx_data,
 			   size_t tx_bytes, bool replace)
 {
 	struct ipc_msg *msg = NULL;
-	uint32_t flags, found = 0;
+	bool found = false;
 	int ret = 0;
+	uint32_t flags;
```

Comment:
Reverse xmas-tree declaration order, flags should be ret.

Reply:
Done.